document-generation/.docx/generateDow.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiBase64SuccessResponse, SuccessStatusCode } from "../../utils/response";
+import { GenerateDocumentRequest, RequestEvent } from "../../models/document-generation";
+import * as fs from "fs";
+import * as path from "path";
+import PizZip from "pizzip";
+import Docxtemplater from "docxtemplater";
+
+export async function generateDow(event: RequestEvent<GenerateDocumentRequest>): Promise<ApiBase64SuccessResponse> {
+  // Extract payload
+  const payload = event.body.templatePayload;
+
+  // Load the docx file as binary content
+  const content = fs.readFileSync(path.resolve(__dirname, "dow.docx"), "binary");
```

Comment:
The `__dirname` property inside of lambda for this file is `/var/task` so the final path ends up being `/var/task/dow.docx`, but the template is stored in `/opt/dow.docx` due to a layer being used for the templates (see [dependencies in a layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path)). 

### Suggestions:
- remove `path.resolve()` and simply provided `/opt/dow.docx` for file path
- use the `getDocxTemplatePath()` in the `utils/utils.ts` to return the file path
- use `path` to create the correct path